pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -31,17 +31,36 @@ local zotero = require('zotero')
 -- -- global state -- --
 local config = {
   client = 'zotero',
-  scannable_cite = false
+  scannable_cite = false,
+  csl_style=''
 }
 
+-- -- -- bibliography marker generator -- -- --
+local function zotero_bibl()
```

Comment:
This function specifically does ODT bibliography

Reply:
> This function specifically does ODT bibliography

Ok I will change the name.